winui/Calendar/Navigation.md

```diff
@@ -56,7 +56,7 @@ sfCalendar.DisplayMode = CalendarDisplayMode.Month;
 {% endhighlight %}
 {% endtabs %}
 
-![restrict-view-navigation-in-winui-calendar](Images/navigation/restrict-view-navigation-in-winui-calendar.gif)
+![restrict view navigation in WinUI Calendar](Images/navigation/winui-calendar-restrict-view-navigation-mode.gif)
```

Comment:
Start with capital letter in Alt text (R)
Remove mode in all places.
![Restrict view navigation in WinUI Calendar](Images/navigation/winui-calendar-restrict-view-navigation.gif)

Reply:
Made changes.